docs/versioned_docs/version-v1.5.0/features/correlation.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "Correlate between HTTP requests/responses via ASP.NET Core middleware"
+layout: default
+---
+
+# Correlation Between HTTP Requests
+
+The `Arcus.WebApi.Correlation` library provides a way to add correlation between HTTP requests. 
+
+This correlation is based the `RequestId` and `X-Transaction-ID` HTTP request/response headers, however, you can fully configure different headers in case you need to.
```

Comment:
```suggestion
This correlation is based on the `RequestId` and `X-Transaction-ID` HTTP request/response headers, however, you can fully configure different headers in case you need to.
```

Reply:
I'll change this in all the versions.